source/chapters/library.rst

```diff
@@ -866,6 +867,86 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
+.. _import-serato-metadata:
+
+Importing metadata from Serato
+------------------------------
+
+From Mixxx 2.3, tracks in your Serato library and crates can now be imported into Mixxx library table.
+Tracks can also be loaded into deck without needing to add the music directory in the preferences. 
+You can also import your Serato library into Mixxx that are in USB drives for a portable Serato library.
+
+**How to import Serato libraries from external USB drive**
+
+* Copy music library from Serato into external USB drive
+* Plug in USB drive while Mixxx is running
+* Click on Serato in the library, all records on USB from Serato will be visible (attach as many as you like!)
```

Comment:
```suggestion
* Click on Serato in the library. All USB drives containing a Serato library will be detected automatically.
```

Reply:
I would work on all the changes you requested immediately and notify you when am done. Thank you so much for your help!